strawberry/aiohttp/test/client.py

```diff
@@ -46,7 +46,8 @@ async def query(
         )
 
         if assert_no_errors:
-            assert resp.status == 200
+            ok_response = 200
+            assert resp.status == ok_response
```

Comment:
I don't like this :P

Reply:
This is the "don't use magic numbers" rule. TBH I'm not a huge fan as well, let's ignore it?